db/re/item_db.txt

```diff
@@ -11969,23 +12157,112 @@
+24475,S_Arm_Shadow_Earring,Arm Shadow Earring,12,,,0,,0,,0,0x00000400,56,2,1048576,,99,1,,{ bonus2 bSkillUseSPrate,"NC_ARMSCANNON",2; .@r = getrefine(); if (.@r >= 2) { bonus2 bSkillUseSPrate,"NC_ARMSCANNON",.@r/2*2; } },{},{}
 //===================================================================
 // New etc.
 //===================================================================
+25000,SPdrainStone_Top,SP Absorption Stone(Upper),6,,,100,,,,,,,,,,,,,{ bonus2 bSPDrainRate,10,1; },{},{}
+25001,DEFStone_Middle,Def Stone(Middle),6,,,100,,,,,,,,,,,,,{ bonus bDef,20; },{},{}
+25002,ChangeLUK_Middle,LUK Exchange Stone(Middle),6,,,100,,,,,,,,,,,,,{ bonus bLuk,3; bonus bStr,-3; },{},{}
+25003,ChangeSTR_Middle,STR Exchange Stone(Middle),6,,,100,,,,,,,,,,,,,{ bonus bStr,3; bonus bInt,-3; },{},{}
+25004,ChangeAGI_Middle,AGI Exchange Stone(Middle),6,,,100,,,,,,,,,,,,,{ bonus bAgi,3; bonus bLuk,-3; },{},{}
+25005,ChangeINT_Middle,INT Exchange Stone(Middle),6,,,100,,,,,,,,,,,,,{ bonus bInt,3; bonus bDex,-3; },{},{}
+25006,ChangeVIT_Middle,VIT Exchange Stone(Middle),6,,,100,,,,,,,,,,,,,{ bonus bVit,3; bonus bAgi,-3; },{},{}
+25007,ChangeDEX_Middle,DEX Exchange Stone(Middle),6,,,100,,,,,,,,,,,,,{ bonus bDex,3; bonus bVit,-3; },{},{}
+25008,ChangeVIT_Bottom,VIT Exchange Stone(Lower),6,,,100,,,,,,,,,,,,,{ bonus bVit,3; bonus bLuk,-3; },{},{}
+25009,ChangeAGI_Bottom,AGI Exchange Stone(Lower),6,,,100,,,,,,,,,,,,,{ bonus bAgi,3; bonus bStr,-3; },{},{}
+25010,ChangeDEX_Bottom,DEX Exchange Stone(Lower),6,,,100,,,,,,,,,,,,,{ bonus bDex,3; bonus bAgi,-3; },{},{}
+25011,ChangeLUK_Bottom,LUK Exchange Stone(Lower),6,,,100,,,,,,,,,,,,,{ bonus bLuk,3; bonus bInt,-3; },{},{}
+25012,ChangeSTR_Bottom,STR Exchange Stone(Lower),6,,,100,,,,,,,,,,,,,{ bonus bStr,3; bonus bDex,-3; },{},{}
+25013,ChangeINT_Bottom,INT Exchange Stone(Lower),6,,,100,,,,,,,,,,,,,{ bonus bInt,3; bonus bVit,-3; },{},{}
+25014,MDEFStone_Bottom,Mdef Stone(Lower),6,,,100,,,,,,,,,,,,,{ bonus bMdef,4; },{},{}
+25015,EXPStone_Bottom,EXP Stone(Lower),6,,,100,,,,,,,,,,,,,{ bonus2 bExpAddClass,Class_All,2; },{},{}
+25016,ATKStone_Bottom,ATK Stone(Lower),6,,,100,,,,,,,,,,,,,{ bonus2 bAddClass,Class_All,1; },{},{}
+25017,MATKStone_Bottom,MATK Stone(Lower),6,,,100,,,,,,,,,,,,,{ bonus bMatkRate,1; },{},{}
 25043,Thorny_Vine_Flute,Thorny Vine Flute,3,0,,0,,,,,,,,,,,,,{},{},{}
 25044,Hard_Thorny_Vine,Hard Thorny Vine,3,0,,10,,,,,,,,,,,,,{},{},{}
 25045,Luxurious_Cloth,Luxurious Cloth,3,0,,0,,,,,,,,,,,,,{},{},{}
 25046,Boarding_Pass,Boarding Pass,3,0,,0,,,,,,,,,,,,,{},{},{}
 25047,Kahlunac,Kahlunac,3,0,,0,,,,,,,,,,,,,{},{},{}
 25048,Hearty_Lunchbox,Hearty Lunchbox,3,0,,0,,,,,,,,,,,,,{},{},{}
 25049,Basilac_Clam,Basilac Clam,3,0,,0,,,,,,,,,,,,,{},{},{}
+25058,TwinkleEffect_Top,TwinkleEffect (Upper),6,,,100,,,,,,,,,,,,,{/*TODO*/},{},{}
```

Comment:
It should be "HAT_EF_LJOSALFAR"

https://www.divine-pride.net/forum/index.php?/topic/3281-archive-kro-cash-shop-items/&do=findComment&comment=4894